presto-delta/src/main/java/com/facebook/presto/delta/DeltaConfig.java

```diff
@@ -46,4 +47,16 @@ public DeltaConfig setMaxSplitsBatchSize(int maxSplitsBatchSize)
         this.maxSplitsBatchSize = maxSplitsBatchSize;
         return this;
     }
+
+    public boolean isCaseSensitivePartitionsEnabled()
+    {
+        return this.caseSensitivePartitionsEnabled;
+    }
+
+    @Config("delta.case-sensitive-partitions-enabled")
```

Comment:
To get a little clarity, was the issue of not being able to query mixed-case/upper-case columns only happening if the column is a partitioned column? Can we also have regular columns as mixed case or upper case, and if yes, does it work as expected?

Reply:
We've experienced the problem only on partitioned columns, as the non partitioned columns are in parquet data and reading parquet the case issue does not happen. However the fix should work on both cases.

Reply:
The reason I wanted to get an idea is we have removed the toLowerCase conversion for all columns not just the partitioned columns. That's why wanted to make sure it does not have any side effects for regular columns and if it's required even for regular columns, should we rename the config to `delta.case-sensitive-columns-enabled`?

Reply:
I can add a test on regular columns with and without the property to be sure

Reply:
I confirm that with `delta.case-sensitive-partitions-enabled=false` uppercase non partitioned columns are properly returning the data, so I think we should leave the property name as it is because it only changes the behavior to partitioned columns. I added the new test that compares the output between a table with a non partitioned column in lowercase and other table with the same column in uppercase to assure that there is no side effect and both tables return the same data.

Reply:
Thanks for adding the tests and confirming.